apiary-metastore-events/kafka-metastore-events/kafka-serializable-events/src/main/java/com/expediagroup/apiary/extensions/events/metastore/common/Preconditions.java

```diff
@@ -19,6 +19,13 @@
 
   private Preconditions() {}
 
+  public static String checkEmpty(String string, String message) {
```

Comment:
This method is more like `checkNotEmpty` and is also has a side effect of trimming the string. I'd suggest splitting this into two separate methods (or do the trim() inline where it's needed, if it actually is necessary).

Reply:
Will split and rename, good spot